src/display/editor/annotation_editor_layer.js

```diff
@@ -151,6 +151,7 @@ class AnnotationEditorLayer {
       case AnnotationEditorType.NONE:
         this.disableTextSelection();
         this.togglePointerEvents(false);
+        this.toggleAnnotationLayerPointerEvents(true);
         this.disableClick();
         break;
```

Comment:
*Slightly off-topic for this PR perhaps, but something that occurred to me:*
Could we replace this with an early return instead, since then we'd not need to check the mode again [further down](https://github.com/mozilla/pdf.js/pull/17702/files#diff-660c5961c3eab2b03e7a0ab7766be0d25e2e040c2423cc663297c830911264b7R176) in this method?
```suggestion
        return;
```